server/service/FileProfileService.js

```diff
@@ -102,71 +97,15 @@ exports.getFileProfileOperation = function(url) {
 }
 
 /**
- * Returns the password for acccessing the file
+ * Configures name of the file
  *
- * uuid String 
- * returns inline_response_200_21
- **/
-exports.getFileProfilePassword = function(url) {
-  return new Promise(async function (resolve, reject) {
-    try {
-      var value = await fileOperation.readFromDatabaseAsync(url);
-      var response = {};
-      response['application/json'] = {
-        "file-profile-1-0:password": value
-      };
-      if (Object.keys(response).length > 0) {
-        resolve(response[Object.keys(response)[0]]);
-      } else {
-        resolve();
-      }
-    } catch (error) {
-      reject();
-    }
-  });
-}
-
-/**
- * Returns the user name for acccessing the file
- *
- * uuid String 
- * returns inline_response_200_20
- **/
-exports.getFileProfileUserName = function(url) {
-  return new Promise(async function (resolve, reject) {
-    try {
-      var value = await fileOperation.readFromDatabaseAsync(url);
-      var response = {};
-      response['application/json'] = {
-        "file-profile-1-0:user-name": value
-      };
-      if (Object.keys(response).length > 0) {
-        resolve(response[Object.keys(response)[0]]);
-      } else {
-        resolve();
-      }
-    } catch (error) {
-      reject();
-    }
-  });
-}
-
-/**
- * Configures path of the file
- *
- * body Fileprofileconfiguration_filepath_body 
+ * body Fileprofileconfiguration_filename_body 
  * uuid String 
  * no response value expected for this operation
  **/
-exports.putFileProfileFilePath = function (url, body) {
-  return new Promise(async function (resolve, reject) {
-    try {
-      console.log(body);
-      await fileOperation.writeToDatabaseAsync(url, body, false);
-      resolve();
-    } catch (error) {
-      reject();
-    }
+exports.putFileProfileFileName = function(body,uuid) {
```

Comment:
For file-profile, we can either create a separate issue for implementing the changes. or we can update the code here in the same issue and merge it to code. 
Kindly let me know your views.

Reply:
implemented FileProfile in this issue only